### PR TITLE
Convert ReactDOMTextarea to createRoot

### PR DESCRIPTION
**Pull request by eps1lon**
Also removes usage of `ReactTestUtils`

### CODE REVIEW

File: packages/react/src/__tests__/codemod.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
+// jscodeshift can take a parser, like "babel", "babylon", "flow", "ts", or "tsx"
```
Comment (rickhanlonii) [inline packages/react/src/__tests__/codemod.js:1]: remove?

**Review by rickhanlonii — APPROVED** [review]
LGTM, just remove the codemod file right?

File: packages/react/src/__tests__/codemod.js (hunk 1 above)
Comment (rickhanlonii) [reply packages/react/src/__tests__/codemod.js:1]: test failures seem to be just from trying to run this as a test

Comment (eps1lon) [reply packages/react/src/__tests__/codemod.js:1]: Whoops 😄

**Pull request merged**